src/i18n/global-messages.js

```diff
@@ -79,6 +79,60 @@ export const txLabels = defineMessages({
   },
 })
 
+export const ledgerMessages = defineMessages({
+  // prerequisites
+  bluetoothEnabled: {
+    id: 'global.ledgerMessages.bluetoothEnabled',
+    defaultMessage: '!!!Bluetooth is enabled on your smartphone.',
+  },
+  locationEnabled: {
+    id: 'global.ledgerMessages.locationEnabled',
+    defaultMessage:
+      '!!!Location is enabled on your device.' +
+      'Android requires location to be enabled to provide access to Bluetooth,' +
+      ' but EMURGO will never store any location data.',
+  },
+  appInstalled: {
+    id: 'global.ledgerMessages.appInstalled',
+    defaultMessage:
+      '!!!Cardano ADA app is installed on your Ledger Nano X device.',
+  },
+  appOpened: {
+    id: 'global.ledgerMessages.appOpened',
+    defaultMessage:
+      '!!!Cardano ADA app must remain open on your Ledger Nano X device.',
+  },
+  internetConnected: {
+    id: 'global.ledgerMessages.internetConnected',
+    defaultMessage: '!!!Your smartphone is connected to the Internet.',
+  },
+  // connection requisites
+  enableTransport: {
+    id: 'global.ledgerMessages.enableTransport',
+    defaultMessage: '!!!Enable bluetooth and location services (Android only).',
```

Comment:
Do we want to specify "Android only" in the string instead of only displaying this on Android devices?